packages/boba/register/addresses/addressesArbitrumMainnet_0x.json

```diff
@@ -1,8 +1,8 @@
 {
   "TK_L1ETH": "0x0000000000000000000000000000000000000000",
-  "TK_L2ETH": "0x0000000000000000000000000000000000000000",
+  "TK_L2ETH": "0x4200000000000000000000000000000000000006",
```

Comment:
0x4200000000000000000000000000000000000006 is optimism stack specific for my knowledge. Arbitrum uses a different stack and 0x0 is used as native coin accordingly. So this change seems to be off. 

Could you elaborate why you think this change needs to be done?